WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -72,16 +74,21 @@ private void StepInputRegistrationPhase()
 				if (round.InputCount < Config.MinInputCountByRound)
 				{
 					Rounds.Remove(round.Id);
+					round.LogInfo($"Round {round.Id} removed because the number of inputs was not enough.");
 				}
 				else
 				{
 					round.SetPhase(Phase.ConnectionConfirmation);
```

Comment:
Here you can see that the round state is actually changing to this. So this should be logged.

Reply:
Should it be logged in Arena.cs or in Round.cs? In Round.cs you can actually log:

```csharp
public void SetPhase(Phase phase)
{
    Logger.LogInfo("Phase changed: {Phase} -> {phase}");
    Phase = phase;
```

Not sure about details here, so this is just an idea to consider.

Reply:
Can be!

Reply:
Done.